src/main/java/duke/task/ToDos.java

```diff
@@ -0,0 +1,33 @@
+package duke.task;
+
+/**
+ * duke.task.ToDos class creates todos
+ * @author Shaune Ang
+ */
+public class ToDos extends Task{
```

Comment:
Should the class be named "ToDo" to match the other Task sub classes (in singular). 
Also there should be a space before '{'.